cmd/account/account-assign.go

```diff
@@ -141,13 +143,15 @@ func (o *accountAssignOptions) run() error {
 
 	if o.payerAccount == defaultPayer {
 		destinationOu = defaultBYOCEnv
```

Comment:
I see that you first create`inputMove` and set its fields`DestinationParentId` to `aws.String(destinationOu)`, and `SourceParentId` to  `aws.String(rootId)`. Then, you check for the different conditions, and you update `destinationOu` but you don't update `inputMove`, so when you pass `inputMove` again to `MoveAccount` it will see the outdated values that you have set initially.